lib/einhorn.rb

```diff
@@ -236,6 +236,20 @@ def self.preload
         else
           log_info("Requiring #{path} (if this hangs, make sure your code can be properly loaded as a library)", :upgrade)
           require path
+
+          if Einhorn::State.config[:gc_for_cow_before_fork]
+            log_info("Starting GC to improve copy-on-write memory sharing")
```

Comment:
Just for general nicety, could we move this into a private helper method named something like `compact_heap`?

Reply:
I went with `force_move_to_oldgen`, because we're not technically compacting. But otherwise done.

Reply:
Good call.